EMacco/abarian-backend
# #169836672 Create migrations

#### What does this PR do?
This PR creates the migrations for the various tables 

#### Description of Task to be completed?
- create migrations for all tables
- create models and associations

#### How should this be manually tested?
- Checkout to this branch 
- Run the command `yarn install`
- Create database called `abarian_db`
- In your `.env` file, add the following fields `DB_USERNAME=`username, `DB_DATABASE=abarian_db`, `DB_HOST=localhost`, `DB_PASSWORD=`password_if_applicable
- Run the command `yarn run migrate`
- All tables should be created. You can confirm using Postico

#### Any background context you want to provide?
- You need to have downloaded and setup `Postgresql` on your local machine

#### What are the relevant pivotal tracker stories?
[#169836672](https://www.pivotaltracker.com/story/show/169836672)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
